script/script.js

```diff
@@ -1,6 +1,13 @@
+/**
+ * @param {number} sum used to save the sum of all the grades
```

Comment:
what is the point behind `@param`?

Reply:
the @param is to identify that 'sum' is a parameter of type {number}.
it is created automatically by vsCode by typing /** 
the correction of the syntax was from this website [link](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html)
specifically the part @param [link](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns)